scripts/create_perf_json.py

```diff
@@ -309,7 +309,7 @@ def get(key: str, counter: bool = False) -> str:
         self.sample_after_value = get('SampleAfterValue')
         self.umask = get('UMask')
         self.unit = get('Unit')
-        self.counter = get('Counter', counter=True)
+        self.counter = get('Counter')
```

Comment:
Maybe rather than use `get` and adding a special case in `get` for `Counter`, here you could just use jd.get('Counter'). get is a convenience routine for the common case.

Reply:
Would the string strip and substring replacement in ```get``` be useful for ```Counter``` field?

Reply:
The strip might. The other substitutions were for off things in descriptions. I suspect Ed has now fixed the ??? thing.

Reply:
Ok, I will change this to use ```jd.get()``` with a strip. Thanks!

Reply:
> The strip might. The other substitutions were for off things in descriptions. I suspect Ed has now fixed the ??? thing.

Correct :). We actively check for trigraphs before pushing event files. Both `???` and `??` are not present in descriptions.

```
perfmon$ egrep -rI '\?{2,}' ./ | grep '/events/'
perfmon$
```